core/src/main/scala/cats/data/Const.scala

```diff
@@ -40,7 +40,8 @@ object Const extends ConstInstances {
   def empty[A, B](implicit A: Monoid[A]): Const[A, B] =
     Const(A.empty)
 
-  final class OfPartiallyApplied[B] {
+/** Uses the [[http://typelevel.org/cats/guidelines.html#partially-applied-type-params Partially Applied Type Params technique]] for ergonomics. */
+  private[data] final class OfPartiallyApplied[B](val dummy: Boolean = true ) extends AnyVal {
```

Comment:
Minor, unnecessarily picky comment: the alignment of the scaladoc comment is a little off here.